astronomer/providers/amazon/aws/operators/emr.py

```diff
@@ -39,6 +42,13 @@ def execute(self, context: Context) -> None:
             configuration_overrides=self.configuration_overrides,
             client_request_token=self.client_request_token,
         )
+        try:
+            # for apache-airflow-providers-amazon>=3.0.0
+            polling_attempts = self.max_tries
```

Comment:
```suggestion
            polling_attempts = self.max_tries  # type: ignore[attr-defined]
```
to be future-safe, so that mypy does not complain for this attribute once the RC is released.

Reply:
Agreed added it

Reply:
if I add this for future-safe, Mypy is complaining Unused "type: ignore" comment

Reply:
I think that should be okay. We can merge this PR once the RC is released. Otherwise, we will need another PR for this fix later